pkg/operator/endpoints/shared.go

```diff
@@ -57,6 +59,7 @@ func RespondIfError(w http.ResponseWriter, err error, strs ...string) bool {
 func RecoverAndRespond(w http.ResponseWriter, strs ...string) {
 	if errInterface := recover(); errInterface != nil {
 		err := errors.CastRecoverError(errInterface, strs...)
+		telemetry.ReportError(err)
```

Comment:
How do you feel about `errors.PrintError(err)` here, and removing the print from `RespondErrorCode()`? The concern is that we do return (instead of panic on) some errors that should be printed (even though most of the errors we return are user errors and shouldn't be printed), so maybe we should make that change when we revisit our internal/user error handling strategy.